src/sql/base/browser/ui/breadcrumb/breadcrumb.component.ts

```diff
@@ -36,6 +36,7 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
 	constructor(
 		@Inject(forwardRef(() => IBreadcrumbService)) private _breadcrumbService: IBreadcrumbService,
 		@Inject(forwardRef(() => Router)) private _router: Router,
+		@Inject(forwardRef(() => ActivatedRoute)) private activeRoute: ActivatedRoute,
```

Comment:
Suggest keeping naming/modifier conventions in the files

`private _activeRoute`

(downside of the current form of suggestions, renaming stuff is annoying if it's used in multiple places....)